bucket/keenwrite.json

```diff
@@ -0,0 +1,36 @@
+{
+    "version": "2.5.1",
+    "description": "Java-based desktop text editor with live preview, string interpolation, and formulas",
+    "homepage": "https://github.com/DaveJarvis/keenwrite",
+    "license": {
+        "identifier": "Unknown",
```

Comment:
I think `Unknown` implies that we cannot find License/EULA of the software, thus leave it unsettled.

Is `Proprietary` applicable to this case?
https://en.wikipedia.org/wiki/Proprietary_software
> While most proprietary software is distributed without the source code, some vendors distribute the source code or otherwise make it available to customers. For example, users who have purchased a license for the Internet forum software vBulletin can modify the source for their own site but cannot redistribute it. This is true for many web applications, which must be in source code form when being run by a web server. The source code is covered by a non-disclosure agreement or a license that allows